src/BaGetter.Azure/BaGetter.Azure.csproj

```diff
@@ -11,14 +11,16 @@
     <PackageReference Include="Azure.Search.Documents" />
     <PackageReference Include="Azure.Storage.Blobs" />
     <PackageReference Include="Microsoft.Azure.Cosmos" />
+    <PackageReference Include="Microsoft.Azure.Cosmos.Table" />
+    <PackageReference Include="Microsoft.Azure.Search" />
+    <PackageReference Include="Microsoft.Azure.Search.Data" />
+    <PackageReference Include="Microsoft.Azure.Search.Service" />
     <PackageReference Include="Microsoft.Extensions.Options.ConfigurationExtensions" />
+    <PackageReference Include="WindowsAzure.Storage" />
```

Comment:
`WindowsAzure.Storage` is deprecated, NuGet gallery is just slow at upgrading it.